rose/web/index.html

```diff
@@ -13,6 +13,7 @@
 
         <button id="start" disabled>Start</button>
         <button id="stop"  disabled>Stop</button>
+        <label id="info"></label>
```

Comment:
This works, but I thought abut something more fancy - instead of rending the track/obstances/cars - render big gray "NO PLAYER CONNECTED". This require much more work, so lets do this later.

If you like to play with this idea, I think it can be done like this:

- in all components, do not render anything if state.players.length == 0
- in a new NoPlayer component, render the the text when state.players.length == 0 and render nothing otherwise.

Another option is to have multiple divs, one for the game canvas, one for "no player connected" message, and show only one div at a time using hidden="true" and hiden="none". The the text can be plain html <h2>No player connected</h2>